docker/zap-api-scan.py

```diff
@@ -392,7 +392,7 @@ def main(argv):
 
         if context_file:
             # handle the context file, cant use base_dir as it might not have been set up
```

Comment:
Did you notice this comment? Might need to check to see if it has been set up...

Reply:
I have added a check and logging to deal with that part. It might be a better solution to just set `base_dir ` up here though: https://github.com/zaproxy/zaproxy/blob/main/docker/zap-baseline.py#L171

What's your opinion on that?